### ISSUE TITLE
It would be cool if `version` supported installation as a deno script

### ISSUE THREAD

**Issue by justinmchase** [dylanpyle/version#7]
Here's the docs:
https://deno.land/manual@v1.11.5/tools/script_installer

The usage would essentially change to:

```sh
deno install -n version -A https://deno.land/x/version/install.ts

version init
version get
```

**Comment by dylanpyle** [comment]
Hey, cool! I actually wasn't aware of that feature, but that's much nicer for repeat usage.

It looks like it may already work! Just tried this and we seem to be up and running, so this might just want a README update.

```
$ deno install -n version -A https://deno.land/x/version@1.0.3/index.ts
Download https://deno.land/x/version@1.0.3/index.ts
Download https://deno.land/x/version@1.0.3/user-error.ts
Download https://deno.land/x/version@1.0.3/git.ts
Check https://deno.land/x/version@1.0.3/index.ts
✅ Successfully installed version
/Users/dylan/.deno/bin/version

$ which version
/Users/dylan/.deno/bin/version

$ version get
1.0.3
```

**Comment by dylanpyle** [comment]
Fixed in #8 , thanks for flagging this!

And, seriously, thanks for all the feedback here @justinmchase. It's great to know it's getting used enough that you have opinions 😁

**Comment by justinmchase** [comment]
Well that was easy!
